automation-tests-docker-entrypoint.sh

```diff
@@ -5,7 +5,9 @@ echo $SBT_CMD
 
 set -o pipefail
 
-sbt -batch -Djsse.enableSNIExtension=false -Dheadless=true "${SBT_CMD}"
+cp -rf src/test/resources/* automation/src/test/resources
```

Comment:
I'm curious why this `cp` is needed... though I'm just starting reviewing this PR.

Reply:
the render config script in firecloud-automation-tests render them under `src/test/resources/*`, but automation being a sub project, we need these config files under `automation/src/test/resources` now